OpenGov-OpenData/ckanext-custom-schema
# [PA-842]Update CDT schema

## [PA-842](https://opengovinc.atlassian.net/browse/PA-842)

## Description

CDT requested that more required metadata fields to be required to match CA minimum metadata requirements. The following fields were made required:
 - Tags
 - Frequency
 - Public Access Level
 - Rights

Along with this, when adding a dataset, there is a dropdown menu to make the dataset you are creating belong to a Topic

## Test Procedure
 
IN THIS DIRECTOY `python setup.py develop`
1. In your ini file, enable `custom_schema`  and `scheming_datasets`. Make sure these come before `og-theme`
2. Go to Dataset page and click Add Dataset
3. Make sure the metadata fields I mentioned in the description are required for creating a dataset.
4. Create dataset.

# Approval Criteria 

You are able to create a dataset and are required to fill out the metadata fields I mentioned in description. Also a dropdown menu for Topic appears while creating a dataset listing the topics in your CKAN.

## Submitter Checklist
